lib/Runtime/Base/JnDirectFields.h

```diff
@@ -140,8 +140,6 @@ ENTRY(every)
 ENTRY(exec)
 ENTRY2(false_, _u("false")) // "false" cannot be an identifier in C++ so using "false_" instead
 ENTRY(flags)
-ENTRY(flat)
-ENTRY(flatMap)
```

Comment:
Drive by cleanup - these Entries were not needed (added when I implemented .flat and .flatMap but not actually needed for JsBuiltin methods)

Reply:
Is this what was causing issues in the legacy win7 and win8 builds?

Reply:
I don't have a windows7/8 setup to test on but this was my best guess - the assertion was saying a property name was invalid so I guessed that the mechanism it was using relied on this ENTRY macros.